README.md

```diff
@@ -34,7 +34,7 @@ python netbox/manage.py dumpdata \
 
 ### Importing the data into Nautobot
 
-From the Nautobot root directory, run `nautobot-server import_netbox_json <json_file> <netbox_version>`, for example `nautobot-server import_netbox_json /tmp/netbox_data.json 2.10.3`.
+As the `nautobot` user account, run `nautobot-server import_netbox_json <json_file> <netbox_version>`, for example `nautobot-server import_netbox_json /tmp/netbox_data.json 2.10.3`.
```

Comment:
Could we instead change this to say "From within the Nautobot application environment" so that it's not exactly tightly coupled to production deployments?
```suggestion
From within the Nautobot application environment, run `nautobot-server import_netbox_json <json_file> <netbox_version>`, for example `nautobot-server import_netbox_json /tmp/netbox_data.json 2.10.3`.
```